ironfish/src/rpc/server.ts

```diff
@@ -2,24 +2,27 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+import { InternalStore } from '../fileStores'
 import { createRootLogger, Logger } from '../logger'
-import { IronfishNode } from '../node'
 import { IRpcAdapter } from './adapters'
-import { ApiNamespace, Router, router } from './routes'
+import { ApiNamespace, RequestContext, Router, router } from './routes'
 
 export class RpcServer {
-  readonly node: IronfishNode
+  readonly internal: InternalStore
+  readonly context: RequestContext
   readonly adapters: IRpcAdapter[] = []
 
-  private readonly router: Router
   private _isRunning = false
   private _startPromise: Promise<unknown> | null = null
   logger: Logger
 
-  constructor(node: IronfishNode, logger: Logger = createRootLogger()) {
-    this.node = node
-    this.router = router
```

Comment:
If you pass a router in here now, and then try to get the server from the router the server will be incorrectly null.